docs/esc_reference.md

```diff
@@ -164,6 +164,9 @@ Items can also change state by playing animations from an `AnimationPlayer` name
 - `walk_block object1 object2`
   Moves object1 towards the position of object2, at the speed determined by object1's "speed" property. This command is blocking.
 
+- `turn object degrees`
+  Turns `object` to a `degrees` angle. The `degrees` value must be between [0, 360] or an error is reported.
```

Comment:
Do you mean this command turns the object for the given `degrees` amount? Or does it specify a target "look angle" (0 = front, 90 = east, 180 = turned back...) as defined in the player_animations.gd file?

Reply:
No, it turns it to a given angle, ie. it specifies the angle.

Let's see if I can make it clearer somehow...